### ISSUE TITLE
Links inside list-group-items collapse surrounding spaces

### ISSUE THREAD

**Issue by parnic** [twbs/bootstrap#23267]
I have some list-group-items in my bootstrap-3 site which contain multiple links inside - these work great. When updating to bootstrap-4 alpha-6, however, I noticed that the spaces around any links in these list-group-items are gone. An example is: https://www.bootply.com/b94xfbq8hc
```html
<ul class="list-group">
  <li class="list-group-item">text to the left - <a href="#">link text</a> - text to the right</li>
</ul>
```
which results in:
![image](https://user-images.githubusercontent.com/888126/29152775-cdd623ba-7d4e-11e7-87ad-ca9da2f56122.png)
The issue is made more obvious when highlighting the text in Chrome 60 as you can see that it's showing overlapping regions:
![image](https://user-images.githubusercontent.com/888126/29152787-e4fcd85e-7d4e-11e7-9643-3890c0c7f21c.png)

(same example in bootstrap-3, for comparison: https://www.bootply.com/JMjwqO6W66)

Is there a way to work around this issue or are embedded links inside list-group-items not supported anymore?

**Comment by mdo** [comment]
Try the latest from `v4-dev`.

**Comment by parnic** [comment]
I'd be happy to. What command do I need to run with the current v4-dev code to compile the css to test that? The docs I've found are no longer accurate and the gruntfile is now mostly empty.

**Comment by Johann-S** [comment]
You can just use the last dist files in the `dist` folder @parnic 😉

**Comment by parnic** [comment]
@Johann-S thanks, that worked! I can confirm that the issue is fixed in at least 1d90f1617.

Any suggestions on what I can do to work around it in alpha-6? I'm using the bootstrap gem for a Rails site currently, though I suppose I could drop in the latest css if necessary.

**Comment by mdo** [comment]
Our next release is coming soon, so you can update to that or try to point the gem to the latest from the branch (unsure if that's doable with RubyGems, it is with Bower/npm).
